geth/node/node.go

```diff
@@ -115,8 +115,11 @@ func defaultEmbeddedNodeConfig(config *params.NodeConfig) *node.Config {
 		nc.HTTPPort = config.HTTPPort
 	}
 
-	if config.BootClusterConfig.Enabled {
-		nc.P2P.StaticNodes = makeBootstrapNodes(config.BootClusterConfig.BootNodes)
+	if config.ClusterConfig.Enabled {
+		// TODO(themue) Should static nodes always be set? Had been done via
```

Comment:
it is true by default and can be false only if node is standalone, right? so it is ok to leave it as it is
anyway i don't think that anyone uses standalone node

Reply:
If standalone is used then `ClusterConfig.Enabled` should be `false`. This fragment looks good.